src/theme/index.ts

```diff
@@ -0,0 +1,10 @@
+import { createTheme } from '@mui/material/styles';
+
+const theme = createTheme({
+  spacing: 8,
+  borders: {
+    solid: (size: number, color?: string) => (color ? `${size}px solid ${color}` : `${size}px solid`),
```

Comment:
what if I'll decide to use rem instead of pixels? 
also you don't need ternary operator here at all